src/test/java/uk/gov/digital/ho/pttg/SimpleHttpClient.java

```diff
@@ -1,17 +1,39 @@
 package uk.gov.digital.ho.pttg;
 
+import org.apache.http.HttpHost;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.AuthCache;
+import org.apache.http.client.CredentialsProvider;
 import org.apache.http.client.methods.CloseableHttpResponse;
 import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.protocol.HttpClientContext;
 import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.auth.BasicScheme;
+import org.apache.http.impl.client.BasicAuthCache;
+import org.apache.http.impl.client.BasicCredentialsProvider;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.apache.http.util.EntityUtils;
 import uk.gov.digital.ho.pttg.api.HttpResponse;
 
 import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
 
 class SimpleHttpClient {
 
+    private final Map<String, String> basicAuthConfig;
```

Comment:
This member could just be an arg to the constructor, with the constructor passing it to the getCredentialsProvider method.

Reply:
Good spot!